.github/workflows/publish.yml

```diff
@@ -0,0 +1,90 @@
+# This workflow will upload a Python Package to Release asset
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Create Release
+
+on:
+  push:
+    branches:
+      - release/*
```

Comment:
A quick question: is there a particular reason for choosing this the trigger? What about making the action triggered whenever a new tag starting with "v" is created? I feel the latter is a bit more straightforward.

Reply:
The main reason for choosing this trigger is since it was easier for me to test with and since I didn't know which trigger you would prefer.
Changing it to trigger on v* tags should not be an issue